config/v1/types_cluster_version.go

```diff
@@ -732,6 +745,30 @@ type Update struct {
 	//
 	// +optional
 	Force bool `json:"force"`
+
+	// acceptRisks is an optional set of names of conditional update risks that are considered acceptable.
+	// A conditional update is performed only if all of its risks are acceptable.
+	// This list may contain entries that apply to current, previous or future updates.
+	// The entries therefore may not map directly to a risk in .status.conditionalUpdateRisks.
+	// acceptRisks must not contain more than 1000 entries.
+	// Only one accept risk is allowed per risk name.
```

Comment:
nit: This sentence is calling for uniqueness, right?  E.g. no `acceptRisks: ["A", "B", "A"]`.  Is there standard language we can use to describe that, like "Entries must be unique" or "Duplicate entries are not allowed" or "A risk may only appear in once in this set"?  It looks like down on the `status` side, we're using "Entries must be unique...".